solutions/USACO open16-bull-in-china-shop.mdx

```diff
@@ -0,0 +1,135 @@
+---
+id: usaco-640
+source: USACO Bronze 2016 Open
+title: Bull in a China Shop
+author: Maggie Liu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_bcs_bronze_open16.html)
+
+## Explanation
```

Comment:
Do we really need this explanation??? This code seems really similar to the official one... The only difference is that this is in C++ and the official is in Java. I think you should remove this.

Reply:
who asked for your review?

Reply:
> who asked for your review?

@jessechoe10 is on content author team

Reply:
It is pretty similar, but the official solution is wrong...

Reply:
How is the official solution wrong?

Reply:
See the warning in the explanation, if it doesn't make sense then I can explain more.